### PR TITLE
mysqldump: add Spanish translation

### PR DESCRIPTION
**Pull request by ferrvic**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/commom/mysqldump.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+> Crea una copia de seguridad de bases de datos MySQL.
+> Vea también `mysql` para restaurar bases de datos.
+> Más información: <https://dev.mysql.com/doc/refman/en/mysqldump.html>.
+
+- Crea un backup (se le pedirá la contraseña al usuario):
+
+`mysqldump --user {{usuario}} --password {{nombre_base_de_datos}} -r {{ruta/al/archivo.sql}}`
+
+- Crea un backup de todas las bases de datos y redirige la salida a un fichero (se le pedirá la contraseña al usuario):
```
Comment (navarroaxel) [inline pages.es/commom/mysqldump.md:11]:
```suggestion
- Crea un backup de todas las bases de datos y redirige la salida a un archivo (se le pedirá la contraseña al usuario):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
@ferrvic Thanks for your submission, but the translations should be made from the primary (English) pages. You can add/edit examples to the primary page in this PR and we can merge the edition and the translation together.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by ferrvic** [comment]
> @ferrvic Thanks for your submission, but the translations should be made from the primary (English) pages. You can add/edit examples to the primary page in this PR and we can merge the edition and the translation together.

Thanks! Changes done!

**Comment by navarroaxel** [comment]
@ferrvic you should remove the last example to match the English page, or add the 3rd example to the English page in this PR.

**Review by schneiderl — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by LukBukkit** [comment]
Oh I spotted a typo in the folder name, instead of commo~m~, it should be common.
Take a look at the page `pages.es` folder: https://github.com/tldr-pages/tldr/tree/master/pages.es

**Pull request merged**